spring-ai-bom/pom.xml

```diff
@@ -212,6 +212,12 @@
             </dependency>
 
             <!-- Spring Boot Starters -->
+            <dependency>
```

Comment:
Please add the `<artifactId>spring-ai-cassandra</artifactId>` to the BOM as well.